tests/Integration/Behaviour/Features/Scenario/Product/Combination/Shop/delete_multishop_combination.feature

```diff
@@ -0,0 +1,263 @@
+
+  Scenario: Delete all combinations one by one from the default shop
+    When I delete combination "product1SWhite" from shop "shop1"
+    And I delete combination "product1SBlack" from shop "shop1"
+    And I delete combination "product1SBlue" from shop "shop1"
+    And I delete combination "product1MWhite" from shop "shop1"
+    And I delete combination "product1MBlack" from shop "shop1"
+    And I delete combination "product1MBlue" from shop "shop1"
+    And product "product1" should have no combinations for shops "shop1"
+    And product "product1" should have the following combinations for shops "shop2":
+      | id reference        | combination name        | reference | attributes           | impact on price | quantity | is default |
+      | product1SWhiteShop2 | Size - S, Color - White |           | [Size:S,Color:White] | 0               | 0        | true       |
+      | product1SBlackShop2 | Size - S, Color - Black |           | [Size:S,Color:Black] | 0               | 0        | false      |
+      | product1SBlueShop2  | Size - S, Color - Blue  |           | [Size:S,Color:Blue]  | 0               | 0        | false      |
+      | product1MWhiteShop2 | Size - M, Color - White |           | [Size:M,Color:White] | 0               | 0        | false      |
+      | product1MBlackShop2 | Size - M, Color - Black |           | [Size:M,Color:Black] | 0               | 0        | false      |
+      | product1MBlueShop2  | Size - M, Color - Blue  |           | [Size:M,Color:Blue]  | 0               | 0        | false      |
+    And product "product1" should not have a default combination for shop "shop1"
+    And product "product1" default combination for shop "shop2" should be "product1SWhiteShop2"
+
+  Scenario: Delete all combinations one by one from all shops
+    When I delete combination "product1SWhite" from shop "shop1"
+    And I delete combination "product1SBlack" from shop "shop1"
+    And I delete combination "product1SBlue" from shop "shop1"
+    And I delete combination "product1MWhite" from shop "shop1"
+    And I delete combination "product1MBlack" from shop "shop1"
+    And I delete combination "product1MBlue" from shop "shop1"
+    And I delete combination "product1SWhiteShop2" from shop "shop2"
+    And I delete combination "product1SBlackShop2" from shop "shop2"
+    And I delete combination "product1SBlueShop2" from shop "shop2"
+    And I delete combination "product1MWhiteShop2" from shop "shop2"
+    And I delete combination "product1MBlackShop2" from shop "shop2"
+    And I delete combination "product1MBlueShop2" from shop "shop2"
+    Then product "product1" should have no combinations for shops "shop1"
+    And product "product1" should have no combinations for shops "shop2"
+    And product "product1" should not have a default combination for shop "shop1"
+    And product "product1" should not have a default combination for shop "shop2"
```

Comment:
"all shops" in dedicated pr i guess. This Pr scope is a bit too big now :smile: (I should have started only with the generating command first, but it is what it is now.. :smile: )

Reply:
Ok, but we risk forgetting it ^^

Reply:
We don't even have the UI prepared for all shops feature of combinations, so I think when time comes we will eventually need to integrate it, I cannot do everything in this one. If I were to handle all shops I see it could grow double the size, it may not be as easy as it sounds :smile: 
Btw we need to think of a way how to get "all the shops" in behat. We will have to retrieve references for shops by they key "all shops". Probably will need an additional assertion step which would collect all the shops smth like "Then I should have following shops: "shop1, shop2, shop3" considered "all shops"

Reply:
As I said in another comment, ok for the next PR that handles:
- generate for all shops
- delete for all shops
- behat tests
- form integration (checkbox in generation modal, double button in the list - temporary UX, checkbox in the bulk deletion)

So this will be the one that is tested by QA

Reply:
For your `all shops` reference key I don't think it's needed we should remain extra explicit about what we assert because `all shops` doesn't mean all the existing shops but all the **associated shops** so adding a magic reference would hide the complexity of the test and assertions

Most of the time we have four shops to test so it's doable to define them explicitely